notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,17 +2,19 @@
  "cells": [
```

Comment:
<p>Can you add some prose about information learned from these (+ other corr matrix) graphs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>We intend to look at correlation plots in detail in https://github.com/aicoe-aiops/ocp-ci-analysis/issues/281, however, can add a sentence on what this means and what is upcoming here :) </p>